public/app/plugins/panel/logs/LogsPanel.tsx

```diff
@@ -22,12 +21,11 @@ export const LogsPanel: React.FunctionComponent<LogsPanelProps> = ({
   }
 
   const newResults = data ? dataFrameToLogsModel(data.series, data.request?.intervalMs, timeZone) : null;
-  const sortedNewResults = sortLogsResult(newResults, sortOrder);
 
```

Comment:
Instead of sorting here, we are passing the `sort order` to `LogsRow` and using the same functionality as in Explore to sort the order of logs.